composer.json

```diff
@@ -18,9 +18,11 @@
     "require": {
         "php": ">=5.5.9",
         "doctrine/common": "~2.4",
+        "fig/link-util": "^1.0",
```

Comment:
`fig/link-util` added back as a dependency of the WebLink component. It fixes the inconsistency, makes the component easier to use for end users and... it's the only one implementation of PSR-13 for now.